etc/profile-m-z/tvnamer.profile

```diff
@@ -0,0 +1,72 @@
+# Firejail profile for tvnamer
+# Description: Automatic TV episode file renamer
+quiet
+# Persistent local customizations
+include tvnamer.local
+# Persistent global definitions
+include globals.local
+
+blacklist /tmp/.X11-unix
+blacklist /usr/libexec
+blacklist ${RUNUSER}
+
+noblacklist ${HOME}/.config/tvnamer
+
+# Allow python (blacklisted by disable-interpreters.inc)
+include allow-python2.inc
+include allow-python3.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-programs.inc
+include disable-proc.inc
+include disable-shell.inc
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/tvnamer
+whitelist ${HOME}/.config/tvnamer
+whitelist ${DOWNLOADS}
+whitelist ${VIDEOS}
+include whitelist-common.inc
+include whitelist-run-common.inc
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+ipc-namespace
+machine-id
```

Comment:
```suggestion
machine-id
netfilter
```

Does this work?

Reply:
Ah yes, I had netfilter in my tvnamer.local and forgot to move it to tvnamer.profile. Fixed!